docs/error-reporting/platform-integrations/real-devices/setup.md

```diff
@@ -0,0 +1,128 @@
+---
+id: setup
+title: Setting Up Backtrace for your Real Device Tests
+sidebar_label: Setup
+description: Link your Sauce Labs real-device tests with Backtrace to view crash and error reports for faster debugging.
+---
+Link your Sauce Labs real-device tests with Backtrace to view crash and error reports for faster debugging. Detecting errors and crashes and resolving them early on is key to the success of your app.
+
+After you’ve completed the steps on this page, crashes and errors associated with your real-device tests (automated or live) will be available for you in Sauce Labs.
```

Comment:
speaking with YuYi today we thin we should tone this down a bit.
Customers will be able to see the number of errors and list of crashes (by name) but not the whole bulk that Backtrace provides.
Might there be an elegant way to say hits without giving a false impression?

Reply:
I think this part is fine. I was thinking that in Line 83 `Once this is set up you will be able to directly view your crashes and errors in Sauce Labs.` might be a bit misleading with the "directly".